airflow/models/dag.py

```diff
@@ -374,6 +375,9 @@ def __init__(
     ):
         from airflow.utils.task_group import TaskGroup
 
+        if tags and any(len(tag) > TAG_MAX_LEN for tag in tags):
+            raise AirflowException("tag cannot be longer than 100 characters")
```

Comment:
Sorry, short question,
should we map the Exception text to the TAG_MAX_LEN variable, too?

Reply:
pls